src/librustc_resolve/diagnostics.rs

```diff
@@ -843,6 +843,17 @@ Or:
 let unknown_variable = 12u32;
 let x = unknown_variable; // ok!
 ```
+
+If the item isn't available in the current module, please check you imported it:
```

Comment:
I think I would word this slightly differently:

"If the item is not defined in the current module, it must be imported using a `use` statement, like so:"